modules/publisher/views.py

```diff
@@ -471,6 +491,8 @@ def post_market_snap(snap_name):
             )
 
         flask.flash("Changes applied successfully.", 'positive')
+    else:
+        flask.flash("No changes to save.", 'information')
```

Comment:
With the little hack I did, once fixed. Test this if it is really useful, because of the redirection the line after

Reply:
Yes, this message may be unnecessary. But at least it gives some message to user when empty changes are posted for some reason.

Reply:
Ok the message stays, I wasn't sure, but it's ok!